lib/orderbook/MatchesProcessor.ts

```diff
@@ -31,6 +31,7 @@ class MatchesProcessor {
 
   public executeSwap(_args) {
     // TODO: execute the swap procedure
+    // TODO: invalidate the maker if it is of the type ownOrder
```

Comment:
This function will probably be used only to instantiate swaps, so the maker order can't be an `ownOrder`, and no invalidation is required. In this case, I think we should remove this comment.